gptel.el

```diff
@@ -402,7 +402,9 @@ Model parameters can be let-bound around calls to this function."
             (set-marker (make-marker) position buffer))))
          (full-prompt
          (cond
-          ((null prompt) (gptel--create-prompt start-marker system))
+          ((null prompt) 
+           (let ((gptel--system-message (or system gptel--system-message)))
```

Comment:
I just realized you don't need `(or system gptel--system-message)` here, just `system` should do since it's set to `gptel--system-message` in the `cl-defun` call.

Reply:
makes sense, thanks. updated